src/assignments/phat/lesson_2_answers.clj

```diff
@@ -1,3 +1,38 @@
 (ns lesson-2-answers)
 
-;;; Enter your answers in here.
+;;;;;;;;;;;;;;;;;
+;; first approach
+
+;; split each string into characters => list char-array
+;; sort list and compare
+;; if two lists equal than add the item to anagram vector
+(defn is_anagram
+  [string1 string2]
+  (if (= (sort (apply list (char-array string1))) 
+         (sort (apply list (char-array string2))))
+    true
+    false))
+
+;; >>> another function to utilise is_anagram
+;; not done
+
+;;;;;;;;;;;;;;;;;;
+;; second solution
+
+;; convert each item into a set
+(defn find_anagram 
+  [vector] 
+  (set 
+   (map set (vals (group-by set vector)))))
```

Comment:
Nice attempt, but unfortunately buggy:
```
(find_anagram ["veer" "lake" "item" "kale" "mite" "ever" "everever"])
;; =>> #{#{"ever" "everever" "veer"} #{"kale" "lake"} #{"item" "mite"}}
```
Can you spot the problem?

Reply:
Yes. I have noticed this problem. 

Thanks for your recommendation above with (sort string), I manage to fix this bug by group-by sorted list of characters.

The latest version of this function would be:

(defn find-anagram 
  [vector] 
  (set 
   (filter #(>= (count %) 2) 
           (set (map set (vals (group-by sort vector)))))))

I will make it in the next pull request.

Best regards